atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -254,19 +278,46 @@ private static SerializableTransactionManager create(
                 transactionService,
                 sweepStrategyManager,
                 cellsSweeper);
-        BackgroundSweeper backgroundSweeper = BackgroundSweeperImpl.create(
-                transactionManager,
+        BackgroundSweeperPerformanceLogger sweepPerfLogger = new NoOpBackgroundSweeperPerformanceLogger();
+        Supplier<SweepBatchConfig> sweepBatchConfig = Suppliers.ofInstance(getSweepBatchConfig(config));
```

Comment:
Yup, the merge conflicts should be resolved carefully. Tagging @fsamuel-bs in case this gets in first :)